persistence/ravendb/index.md

```diff
@@ -29,7 +29,7 @@ Specific versions of RavenDB Persistence are tied to a major version of NService
 |     5.x     |        3.0.x        |       3.0      | .NET 4.5.2  |
 |     5.x     |         2.x         |       2.5      | .NET 4.5.2  |
 
-NOTE: .NET Core is currently not a supported configuration.
+NOTE: .NET Core is currently a not supported platform.
```

Comment:
@mauroservienti that reads strange...

Reply:
"an unsupported platform" rather than "a not supported platform" is what @mauroservienti was going for I think. Updated, and will merge once the checks allow.